index.bs

```diff
@@ -55,7 +55,7 @@ urlPrefix: https://html.spec.whatwg.org/multipage/
             text: expose a user interface to the user
             text: text track cue order
         urlPrefix: syntax.html
-            text: syntax-charref
+            text: character references; url: #syntax-charref
```

Comment:
Bikeshed wouldn't build otherwise. It's unrelated so doesn't belong here. I can fix separately and rebase...